packages/react/src/section-alert/docs/code.mdx

```diff
@@ -1,3 +1,126 @@
 ## Props
 
 <ComponentPropsTable name="SectionAlert" />
+
+## Focusing the alert
+
+_NOTE: When focusing an alert after closing a `Drawer`, use the [focusElementOnClose prop in the Drawer component](/components/drawer#focus-custom-element-on-close)._
+
+In the event you need to focus the alert, you can do so in one of three ways:
+
+1. [Automatically focus on mount](#automatically-focus-on-mount).
+2. [Automatically focus on update](#automatically-focus-on-update).
+3. [Manually set focus](#manually-set-focus).
+
+### Automatically focus on mount
+
+Press the "Toggle alert" button below to mount/unmount the alert. Notice that each time the alert is mounted it will be focused. Here we use the `focusOnMount` boolean prop to automatically handle the focusing.
```

Comment:
```suggestion
Press the “Toggle alert” button below to mount/unmount the alert. Notice that each time the alert is mounted it will be focused. Here we use the `focusOnMount` boolean prop to automatically handle the focus management.
```